appveyor.yml

```diff
@@ -41,21 +41,22 @@ after_test:
     - "SET PATH=C:\\Python34;C:\\Python34\\Scripts;%PATH%"
     - pip install codecov
     - codecov -f "SparkCLRCodeCoverage.xml"
-    - cmd: cd .\build
+    - cmd: cd .\build\localscript
     - cmd: .\Runsamples.cmd --validate
-    - cmd: cd ..
+    - cmd: cd ..\..
     - cmd: dir csharp\Microsoft*.nupkg
     - cmd: dir scala\target\spark-clr*.jar
 
 artifacts:          
   - path: '**\Microsoft*.nupkg'                 # find all NuGet packages recursively
   - path: 'scala\target\spark-clr*.jar'
+  - path: 'build\target\spark-clr*.zip'
 
 deploy:
   - provider: GitHub                            # deploy to GitHub releases
     auth_token:                                 
       secure: 4JIzL2DO3OXXjdrso+VhA0RjMvUAg8BnZRHXojswaryXFe9mDJ3ExjssASWL1FZq 
-    artifact: /Microsoft.*\.nupkg/,/spark-clr.*\.jar/
+    artifact: /spark-clr.*\.zip/
```

Comment:
Initially, Wiki page is lot easier to adjust and make constant changes. Once Wiki page stabilized, agreed, we should convert wiki page content into a readme.md or some sort.